### PR TITLE
Fix battery status and optimize and fix status pull

### PR DESCRIPTION
**Pull request by haslinghuis**
LiveStatusUpdate stops intermittent. This PR fixes the update and removes duplicate status pulls optimizing (serial) communication.

- [x] FC.ANALOG was wrongly assigned with FC.INITIAL_CONFIG
- [x] Update date.now to performance.now
- [x] Increase timer for status polling from 100 to 250ms
- [x] Preset tabs was blocking the timer
- [x] Use setInterval instead of restarting GUI.timeout_add() which failed intermittent.
- [x] Cleanup of status polls as it's running consistent in serial backend.

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

File: src/js/msp/MSPHelper.js
Comment (McGiverGim) [inline src/js/msp/MSPHelper.js:314]: I think `performance.now()` is relative to the page load, while `Date.now()` to the UNIX epoch. To be used by a timestamp is good to use the `performance` one?

Comment (haslinghuis) [reply src/js/msp/MSPHelper.js:314]: Double checked, both work, but performance.now() seems to be preferred.

Comment (haslinghuis) [reply src/js/msp/MSPHelper.js:314]: Replaced the GUI.timeout_add() - now the `liveDataRefreshTimer` keeps running :tada:

Comment (haslinghuis) [reply src/js/msp/MSPHelper.js:314]: Also now it runs perfectly all the time we can remove pulling from some tabs to optimize serial communication

**Review by nerdCopter — APPROVED** [review]

**Comment by haslinghuis** [comment]
Rebased on master

**Review by nerdCopter — APPROVED** [review]

**Review by blckmn — APPROVED** [review]

**Comment by HThuren** [comment]
@haslinghuis Great update, I really experience faster execution, well done

**Pull request merged**
